flask_ask/core.py

```diff
@@ -198,7 +183,7 @@ def session_ended(self, f):
 
         @ask.session_ended
         def session_ended():
-            return "{}", 200
+            return question(), 200
```

Comment:
to me, it makes more sense to return empty json here rather than question. a lot of people have requested returning question here, but i've always thought empty json was sufficient. what does returning question enhance?